Description: Disable the architecture verification which is limited to i386 and amd64
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: no
Taken from: https://sources.debian.org/patches/openjfx/11.0.2+1-1/06-disable-architecture-verification.patch
--- a/build.gradle
+++ b/build.gradle
@@ -302,6 +302,7 @@
 // at present building on PI is not supported, but we would only need to make
 // some changes on assumptions on what should be built (like SWT / Swing) and
 // such and we could probably make it work.
+/*
 if (!IS_MAC && !IS_WINDOWS && !IS_LINUX) fail("Unsupported build OS ${OS_NAME}")
 if (IS_WINDOWS && OS_ARCH != "x86" && OS_ARCH != "amd64") {
     fail("Unknown and unsupported build architecture: $OS_ARCH")
@@ -310,6 +311,7 @@
 } else if (IS_LINUX && OS_ARCH != "i386" && OS_ARCH != "amd64") {
     fail("Unknown and unsupported build architecture: $OS_ARCH")
 }
+*/
 
 
 // Get the JDK_HOME automatically based on the version of Java used to execute gradle. Or, if specified,

